doc/pages/faq.asciidoc

```diff
@@ -269,6 +269,9 @@ order to select data in a line without any trailing newline characters, one coul
 use the `[^\n]+` pattern, which is arguably a good compromise when
 balanced against the ability to select data over several lines.
 
+You can instruct the regex engine to stop matching newline characters with
+`.` by disabling the appropriate flag (`(?S)`).
```

Comment:
I didn't want to be too specific because flags can currently be enabled/disabled anywhere in a pattern (not just at the beginning).

I'd like the FAQ to be just pointing fingers at topics to look into, rather than instructions that might become invalid in the future (not that these specific regex details have any real chances to be).

Reply:
I agree, no need for details here so let's keep it.